src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1161,12 +1161,12 @@ public async Task LoadHostProfilesAsync()
             if (this.profilePaths != null)
             {
                 // Load any of the profile paths that exist
+                var command = new PSCommand();
                 foreach (var profilePath in GetLoadableProfilePaths(this.profilePaths))
                 {
-                    PSCommand command = new PSCommand();
-                    command.AddCommand(profilePath, false);
-                    await this.ExecuteCommandAsync<object>(command, sendOutputToHost: true, sendErrorToHost: true).ConfigureAwait(false);
+                    command.AddCommand(profilePath, false).AddStatement();
                 }
+                await ExecuteCommandAsync<object>(command, sendOutputToHost: true, sendErrorToHost: true).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this default value assigned to parameter 'sendErrorToHost'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4896216)